packages/runtime/src/widgets/Form/MultiSelect.tsx

```diff
@@ -40,7 +40,7 @@ export type MultiSelectProps = {
   labelKey?: string;
   valueKey?: string;
   items?: Expression<SelectOption[]>;
-  onItemSelect?: EnsembleAction;
```

Comment:
Let's keep this around but mark it as deprecated. So we will prefer onChange if it's there but fallback to onItemSelect.